### PR TITLE
fix non standard ERC20 approve

### PR DESCRIPTION
**Pull request by steffenix**
I can remove few lines from the script by using deployed contracts instead of redeploying everything.
I could also remove the Tether contract if you are ok to add `hardhat-etherscan-abi`

### CODE REVIEW

File: scripts/fork/usdtDeposit.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    const overrides = { gasLimit: 9500000 }
+    let wallet, wallet2, wallet3, wallet4
+    let usdt
+    let prizePool
+    let poolWithMultipleWinnersBuilder;
+    let compoundPrizePoolABI;
+
+    [wallet, wallet2, wallet3, wallet4] = await hardhat.ethers.getSigners()
+
+    usdt = await ethers.getContractAt('TetherToken', '0xdAC17F958D2ee523a2206206994597C13D831ec7')
```
Comment (asselstine) [inline scripts/fork/usdtDeposit.js:24]: It appears that you only use the `approve()` and `transfer()` functions for the USDT token.  Those are both part of ERC20, so you could reuse the standard ERC20 abi:

```javascript
usdt = await ethers.getContractAt('IERC20Upgradeable', '0xdAC17F958D2ee523a2206206994597C13D831ec7')
```

And then you can remove the `TetherToken.sol`.

**Review by asselstine — CHANGES_REQUESTED** [review]
Good job @steffenix!  Nearly there.  We just need to ensure all test pass: make sure both `yarn test` and `yarn coverage` run without fail.

Coverage must be the same or higher, although I don't think that'll be a problem because it's just one line.

**Comment by steffenix** [comment]
I am having issues with tests, master seems to be broken too but I can't read the report on circle ci.

**Comment by asselstine** [comment]
It appears Coveralls doesn't report broken builds!

- I've added a CircleCI badge, which reports test failures correctly
- I've fixed the tests in the master branch

Please rebase onto the new master!

Thanks for your patience

**Comment by steffenix** [comment]
rebased, all tests are green

**Comment by asselstine** [comment]
I can confirm; all tests pass.  Thanks @steffenix!  Good work.

**Pull request merged**